addons/source-python/packages/source-python/translations/strings.py

```diff
@@ -217,12 +217,6 @@ def _replace_escaped_sequences(given_string):
         # Return the replaced string
         return given_string
 
-    def get_strings(self, key, **tokens):
```

Comment:
Well, if you want to prevent in-place updates, you would change the ``__getitem__`` method. But actually, that doesn't really matter, because you always pass all tokens, right?